tests/qgis-projects/tests/base_layers.qgs.cfg

```diff
@@ -17,14 +17,15 @@
             "5413731.37831568531692028"
         ],
         "mapScales": [
-            10000,
+            1000,
+            5000,
             25000,
             50000,
             100000,
             250000,
             500000
         ],
-        "minScale": 10000,
+        "minScale": 500,
```

Comment:
`minScale` is not equal to `mapScales` minimum?

Reply:
Thnaks for the review